src/app/search/search-notebook-table/search-notebook-table.component.html

```diff
@@ -54,19 +50,11 @@
         </div>
       </mat-cell>
     </ng-container>
-    <ng-container matColumnDef="projectLinks">
-      <mat-header-cell fxShow fxHide.lt-md *matHeaderCellDef>Links</mat-header-cell>
-      <mat-cell fxShow fxHide.lt-md *matCellDef="let entry">
-        <a [href]="entry?.source.providerUrl" *ngIf="entry?.source.providerUrl">
-          <fa-icon class="fa-lg" [icon]="entry.source.providerIcon" [matTooltip]="entry?.source.provider"></fa-icon>
-        </a>
-      </mat-cell>
-    </ng-container>
     <ng-container matColumnDef="starredUsers">
       <mat-header-cell *matHeaderCellDef mat-sort-header start="desc">Stars</mat-header-cell>
-      <mat-cell class="description-cell" *matCellDef="let notebook"
-        >{{ !notebook?.source.starredUsers || notebook?.source.starredUsers.length === 0 ? '' : notebook?.source.starredUsers?.length }}
-        <mat-icon class="star-icon" *ngIf="notebook?.source.starredUsers?.length > 0">star_rate</mat-icon>
+      <mat-cell class="starred-cell weight-bold" *matCellDef="let notebook"
+        ><mat-icon class="star-icon" *ngIf="notebook?.source.starredUsers?.length > 0">star_rate</mat-icon>
+        {{ !notebook?.source.starredUsers || notebook?.source.starredUsers.length === 0 ? '' : notebook?.source.starredUsers?.length }}
```

Comment:
Nit: you don't need the question marks on the far right `notebook?.source.starredUsers?.length` because you already checked they exist earlier in the expression. It doesn't hurt obviously, but it makes it slightly less readable, to me, because I was trying to figure out why the checks were there.

Reply:
This is code I copied/adapted from the workflows template because the previous code didn't format consistently (when I first wrote it, I probably copied it from the wrong place).   Since it's a tiny change, I will make it in each of the templates, to be consistent.  Could be written a lot better, though.

Makes me pine for a framework which makes it easy to define functions that generate little HTML snippets that you can use wherever.  Part of the reason this code isn't already in a pipe is because creating and deploying an Angular pipe is a pain in the butt, relatively speaking...